### DISCUSSION TITLE
Decoupled Orchard Core CMS for QTA with Hisham Bin Ateya - Orchard Core Pair Programming by Lombiq

### DISCUSSION THREAD

**Discussion by Piedone** [OrchardCMS/OrchardCore#16185] [Show and tell]
Join this session of Orchard Core Pair Programming by Lombiq! In these, we do an hour of pair programming with an Orchard Core community member about a project of theirs. We learn together a lot, share best practices, and write some good code. All this is live, and you can join us with your questions! Check it out here: https://www.youtube.com/watch?v=EC8C4tHpxnE. See you at 5 PM UTC on the 3rd!

@hishamco, a core contributor of Orchard Core, joins us with a decoupled CMS project he was building for Quality Training Academy for Health Training from Saudi Arabia. We'll look into the basics first, how you'd start with decoupled Orchard Core. Next, we'll check out more of it in action:

- Decoupled CMS using Razor Pages/MVC
- Modularity
- We could explore some other features like import content, taxonomies, etc. We'll see!

Your host and the "navigator" of the pair programming session will be me.

Would you like to be our guest? Just write a comment here!

**Comment by hishamco** [comment]
Hi everyone, please join us next week on the stream we will do a decoupled OC CMS on action, for a project that I created last year. If you have any suggestions or ideas to add feel free to share and we will see how it goes

**Reply by Piedone** [reply]
What's a secret that only those people who join the stream will know about? :)

**Reply by hishamco** [reply]
We might engage the community before each stream to give some questions or points that might be related to the session, otherwise the people who join will win :)

**Comment by Piedone** [comment]
We're starting in five minutes! Tune in here: https://www.youtube.com/watch?v=EC8C4tHpxnE
